.ci/test.cmd

```diff
@@ -0,0 +1,3 @@
+python.exe -c "from PIL import Image"
+IF ERRORLEVEL 1 EXIT /B
+python.exe -bb -m pytest -v -x -W always --cov PIL --cov Tests --cov-report term --cov-report xml Tests %REVERSE%
```

Comment:
```suggestion
python.exe -bb -m pytest -v -x -W always --cov PIL --cov Tests --cov-report term --cov-report xml Tests
```
REVERSE is only set in https://github.com/python-pillow/Pillow/blob/0f47ecd4325268156c27fa86799d31829f31fcea/.github/workflows/test.yml#L52